cmd/cody-gateway/internal/httpapi/handler.go

```diff
@@ -230,6 +234,18 @@ func NewHandler(
 			otelhttp.WithPublicEndpoint(),
 		),
 	)
+
+	var attributionClient graphql.Client
+	if config.EnableAttributionSearch {
+		attributionClient = dotcomClient
+	}
+	v1router.Path("/attribution").Methods(http.MethodPost).Handler(
+		instrumentation.HTTPMiddleware("v1.attribution",
+			authr.Middleware(
+				attribution.NewHandler(attributionClient, logger),
+			),
+		),
```

Comment:
```suggestion
		),
		otelhttp.WithPublicEndpoint(),
```
Like the other handlers, admittedly I'm not 100% sure on this, but appears to be related to linking traces to the original sg requests

Reply:
`otelhttp.WithPublicEndpoint` is the right configuration for all Cody Gateway endpoints 👍